### DISCUSSION TITLE
E6AD8A0D-D529-470E-9C3C-418F3FEC5FB2

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#98] [General]
# Periodic Comets: 2P to 8P

This addon adds 7 periodic comets as part of a periodic comets catalog.

https://celestia.mobi/addon?item=E6AD8A0D-D529-470E-9C3C-418F3FEC5FB2&lang=zh_CN

**Comment by 13940951572** [comment]
下载完这个插件怎么怎么搜索不到这个插件

**Comment by 13940951572** [comment]
我问一下有一个彗星插件下载完之后重启软件怎么不显示呢？
